runtime/src/serde_snapshot.rs

```diff
@@ -126,6 +141,7 @@ where
         .deserialize_from::<R, T>(reader)
 }
 
+#[cfg(test)]
 #[allow(clippy::too_many_arguments)]
 pub(crate) fn bank_from_stream<R>(
```

Comment:
This function is now only used in serde_snapshots/tests.rs. I could remove it and update the test, but figured it was fine to leave it. Let me know if it should be removed instead.

Reply:
yeah looks like this is just a convenience function around passing a `SnapshotStreams` at this point. I think this can be removed and the functionality just moved inside the one test

Reply:
OK will do!

Reply:
Done!